Documentation/Upgrade/rook-upgrade.md

```diff
@@ -50,18 +51,24 @@ those releases.
     official releases. Builds from the master branch can have functionality changed or removed at any
     time without compatibility support and without prior notice.
 
-## Breaking changes in v1.16
+## Breaking changes in v1.17
 
-* The minimum supported Kubernetes version is v1.27.
+* The minimum supported Kubernetes version is v1.27
 
-* Rook no longer supports Ceph v17 (Quincy) as it is end of life. Rook v1.15 clusters should
-    [upgrade Ceph](./ceph-upgrade.md) to at least Ceph v18 (Reef) before upgrading Rook to v1.16.
-    Be aware of known [Ceph v18.2.4 ARM issues](https://github.com/rook/rook/issues/14502).
+* S3 users provisioned via CephObjectStoreUser resources no longer allow multiple credentials to
+    exist on underlying S3 users. Rook will purge all but one of the credentials. This could be a
```

Comment:
```suggestion
    exist on underlying S3 users, unless explicitly managed by Rook. Rook will purge all but one of the undeclared credentials. This could be a
```
?

This statement makes it sound like multiple credentials/keys aren't possible but I'm not sure what the best phrasing would be.

Reply:
Done